modules/edu.gemini.seqexec.server/src/main/resources/Gmos.xml

```diff
@@ -0,0 +1,660 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<Records xmlns="http://www.gemini.edu/CaSchema">
+    <Top name="tcs">tcs</Top>
+    <Top name="gm">gm</Top>
+    <Top name="ag">ag</Top>
+    <Apply name="gmos::apply">
+        <top>gm</top>
+        <apply>apply</apply>
+        <car>applyC</car>
+        <description>IS Primary Apply Record </description>
+        <command name="gmos::config">
+            <description>Setup a Configuration</description>
+            <parameter name="disperserMode">
+                <channel>grSelect.A</channel>
+                <type>STRING</type>
+                <description>Disperser Mode</description>
+            </parameter>
+            <parameter name="disperser">
+                <channel>grSelect.B</channel>
+                <type>STRING</type>
+                <description>Disperser</description>
+            </parameter>
+            <parameter name="stageMode">
+                <channel>dtaTrack.A</channel>
+                <type>STRING</type>
+                <description>Translation Stage Mode</description>
+            </parameter>
+            <!--
+            <parameter name="useElectronicOffsetting">
+                <channel>wfs:followA.K</channel>
+                <type>INTEGER</type>
+                <description>Electronic Offsets</description>
+            </parameter>
+            -->
```

Comment:
The XML file has limitations. One is that is assumes all parameters are inputs to a CAD. They have a type, but the underlining channel is always of type String. The problem comes is in cases like this, where the parameter type is actually the channel type. The library supports this case, but it needs an additional parameter that is not included in the XML. Therefore, I need to create the corresponding Attribute by hand. I kept definition in the XML as a reminder.